### PR TITLE
Hotfix/base template

### PR DESCRIPTION
**Pull request by VirginiaDooley**
Ref https://github.com/DemocracyClub/dc_django_utils/pull/29 and https://github.com/DemocracyClub/Website/issues/394

This work integrates changes in the base template from `dc-utils`, bringing back the default structure that existed with `dc_base_theme` before we changed to `dc_design_system`.

Check Browserstack shows dc logo favicon: 

- [ ] IE9
- [x] IE 11
- [x] Chrome
- [x] Firefox
- [ ] Safari

Check link previews: all pages should have a logo with the expection of the blog and report page, both which preview a hero image. 

Post page
- [x] https://cards-dev.twitter.com/validator
- [x] Facebook share debugger dev tools
- [x] https://www.linkedin.com/post-inspector

Report page
- [x] https://cards-dev.twitter.com/validator
- [x] Facebook share debugger dev tools
- [x] https://www.linkedin.com/post-inspector 

Other pages
- [x] https://cards-dev.twitter.com/validator
- [x] Facebook share debugger dev tools
- [x] https://www.linkedin.com/post-inspector

I created a new logo file specifically to meet the demands of rectangular og posts such as fb and linkedin. Although it was discussed in slack that it is unlikely we will share certain pages on certain social media platforms, it is possible that all pages are shared on all platforms and the default should be one we are happy with. 

By adding white space around a high-quality version of the logo, the majority of the icon is visible (slight cropping but can be fixed further with additional white space/reduced logo size). If acceptable, I can adopt it into the next version of dc_utils to be used throughout for og images on non blog, person or report pages. 
og_image_logo

Common use of the revised logo will come in the next version of `dc_utils` https://github.com/DemocracyClub/dc_django_utils/issues/30

### CODE REVIEW

**Comment by VirginiaDooley** [comment]
[og tags in dc_utils](https://trello.com/c/oyrIDWEA/3048-og-tags-in-dcutils)

**Comment by symroe** [comment]
I like using BS to parse the hero image, but it felt a little fragile to me. I started thinking about how this might be more explicit, and tried using the `meta` plugin. Here's a patch for using that, for feedback:

```patch
diff --git democracy_club/apps/core/report_helpers/views.py democracy_club/apps/core/report_helpers/views.py
index f9f3a2d..04ccf9a 100644
--- democracy_club/apps/core/report_helpers/views.py
+++ democracy_club/apps/core/report_helpers/views.py
@@ -2,7 +2,6 @@ import markdown
 from django.conf import settings
 from django.template import Template, RequestContext
 from django.views.generic import TemplateView
-from bs4 import BeautifulSoup as bs
 
 
 class MarkdownFileView(TemplateView):
@@ -19,21 +18,17 @@ class MarkdownFileView(TemplateView):
 
         template = Template(self.markdown_content())
         rendered_template = template.render(RequestContext(self.request))
-        md = markdown.Markdown(extensions=settings.REPORT_MARKDOWN_EXTENSIONS)
-        html = md.convert(rendered_template)
-        context["html_content"] = html
-        context["toc"] = md.toc
-        context["page_title"] = md.toc_tokens[0]["name"]
+        self.md = markdown.Markdown(extensions=settings.REPORT_MARKDOWN_EXTENSIONS)
+        self.html = self.md.convert(rendered_template)
+        context["html_content"] = self.html
+        context["toc"] = self.md.toc
+        context["page_title"] = self.md.toc_tokens[0]["name"]
+        context["report_metadata"] = self.md.Meta
         context["report_hero_image"] = self.report_hero_image()
+
         return context
 
     def report_hero_image(self):
-        soup = bs(self.markdown_content(), "html.parser")
-        images = []
-        for img in soup.find_all("img"):
-            img_url = img.attrs.get("src")
-            img_str = img_url.strip("{% static '").strip("' %}")
-            img_path = ("/static/") + img_str
-            if img_url:
-                images.append(img_path)
-        return images[0] if images else None
+        hero_image = self.md.Meta.get("hero_image", None)
+        if hero_image:
+            return hero_image[0]
diff --git democracy_club/apps/report_2022/report.md democracy_club/apps/report_2022/report.md
index cfffcf0..d2c1d5b 100644
--- democracy_club/apps/report_2022/report.md
+++ democracy_club/apps/report_2022/report.md
@@ -1,4 +1,6 @@
-{% load static %}
+{% load static %}---
+hero_image: {% static "images/maze_illustration_no_text.png" %}
+---
 
 # Local and Devolved Elections, 5 May 2022
 
diff --git democracy_club/settings/base.py democracy_club/settings/base.py
index 484f3b5..8a12ff8 100644
--- democracy_club/settings/base.py
+++ democracy_club/settings/base.py
@@ -199,7 +199,7 @@ MARKDOWN_DEUX_STYLES = {
     },
 }
 
-REPORT_MARKDOWN_EXTENSIONS = ["toc"]
+REPORT_MARKDOWN_EXTENSIONS = ["toc", "meta"]
 
 
 def blog_markdown(value):
```

**Review by symroe — COMMENTED** [review]
This is looking good to me! Let's talk about the patch I left above.

After that, I think all that's needed is a fixup and merge :+1:

**Comment by VirginiaDooley** [comment]
> I like using BS to parse the hero image, but it felt a little fragile to me. I started thinking about how this might be more explicit, and tried using the `meta` plugin. Here's a patch for using that, for feedback:
> 
> ```diff
> diff --git democracy_club/apps/core/report_helpers/views.py democracy_club/apps/core/report_helpers/views.py
> index f9f3a2d..04ccf9a 100644
> --- democracy_club/apps/core/report_helpers/views.py
> +++ democracy_club/apps/core/report_helpers/views.py
> @@ -2,7 +2,6 @@ import markdown
>  from django.conf import settings
>  from django.template import Template, RequestContext
>  from django.views.generic import TemplateView
> -from bs4 import BeautifulSoup as bs
>  
>  
>  class MarkdownFileView(TemplateView):
> @@ -19,21 +18,17 @@ class MarkdownFileView(TemplateView):
>  
>          template = Template(self.markdown_content())
>          rendered_template = template.render(RequestContext(self.request))
> -        md = markdown.Markdown(extensions=settings.REPORT_MARKDOWN_EXTENSIONS)
> -        html = md.convert(rendered_template)
> -        context["html_content"] = html
> -        context["toc"] = md.toc
> -        context["page_title"] = md.toc_tokens[0]["name"]
> +        self.md = markdown.Markdown(extensions=settings.REPORT_MARKDOWN_EXTENSIONS)
> +        self.html = self.md.convert(rendered_template)
> +        context["html_content"] = self.html
> +        context["toc"] = self.md.toc
> +        context["page_title"] = self.md.toc_tokens[0]["name"]
> +        context["report_metadata"] = self.md.Meta
>          context["report_hero_image"] = self.report_hero_image()
> +
>          return context
>  
>      def report_hero_image(self):
> -        soup = bs(self.markdown_content(), "html.parser")
> -        images = []
> -        for img in soup.find_all("img"):
> -            img_url = img.attrs.get("src")
> -            img_str = img_url.strip("{% static '").strip("' %}")
> -            img_path = ("/static/") + img_str
> -            if img_url:
> -                images.append(img_path)
> -        return images[0] if images else None
> +        hero_image = self.md.Meta.get("hero_image", None)
> +        if hero_image:
> +            return hero_image[0]
> diff --git democracy_club/apps/report_2022/report.md democracy_club/apps/report_2022/report.md
> index cfffcf0..d2c1d5b 100644
> --- democracy_club/apps/report_2022/report.md
> +++ democracy_club/apps/report_2022/report.md
> @@ -1,4 +1,6 @@
> -{% load static %}
> +{% load static %}---
> +hero_image: {% static "images/maze_illustration_no_text.png" %}
> +---
>  
>  # Local and Devolved Elections, 5 May 2022
>  
> diff --git democracy_club/settings/base.py democracy_club/settings/base.py
> index 484f3b5..8a12ff8 100644
> --- democracy_club/settings/base.py
> +++ democracy_club/settings/base.py
> @@ -199,7 +199,7 @@ MARKDOWN_DEUX_STYLES = {
>      },
>  }
>  
> -REPORT_MARKDOWN_EXTENSIONS = ["toc"]
> +REPORT_MARKDOWN_EXTENSIONS = ["toc", "meta"]
>  
>  
>  def blog_markdown(value):
> ```

Clever! Agree, much neater and more secure. I tried extracting the hero image using `toc` and when that failed, searched options for parsing html. Now I realise I was researching solutions for the wrong problem.

File: Pipfile [hunk 1]
```diff
@@ -33,6 +33,7 @@ regex = "==2022.4.24"
 textile = "==4.0.2"
 mkdocs = "==1.3.0"
 Flask = "==2.1.0"
+beautifulsoup4 = "==4.11.1"
```
Comment (symroe) [inline Pipfile:36]: I think you can remove this now?

**Pull request merged**